File: learning/tyama_paizalearning~fizz-buzz.cs
using System;
class FizzBuzz{
	static void Main(){
		int n=int.Parse(Console.ReadLine());
		for(int i=1;i<=n;i++){
			Console.WriteLine(i%15==0?"Fizz Buzz":i%3==0?"Fizz":i%5==0?"Buzz":i.ToString());
		}
	}
}

File: learning/tyama_paizalearning~fizz-buzz.kt
fun main(args: Array<String>) {
	val n=readLine()!!.toInt()
	for(i in 1..n){
		println(if(i%15==0)"Fizz Buzz" else if(i%3==0)"Fizz" else if(i%5==0)"Buzz" else i)
	}
}
